src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -2,52 +2,66 @@
 
 import com.kodilla.ecommercee.domain.dto.CartDto;
 import com.kodilla.ecommercee.domain.dto.CartItemDto;
-import lombok.RequiredArgsConstructor;
+import com.kodilla.ecommercee.domain.dto.OrderDto;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
-
 import java.math.BigDecimal;
+import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.List;
 
+
 @RestController
 @CrossOrigin("*")
-@RequestMapping(consumes = MediaType.APPLICATION_JSON_VALUE, value = "/carts")
+@RequestMapping("/carts")
 public class CartController {
     
-    @PostMapping(value = "/createCart/{userId}")
-    public ResponseEntity<Void> createCart(@PathVariable Long userId) {
-        List<CartItemDto> cart = new ArrayList<>();
-        //post to repository//
-        return ResponseEntity.ok().build();
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I am thinking if this method needs getting JSON or we can simplify it by @PathVariable or @RequestParam with userId? What do you think?

Reply:
What would you say for this:
    @PostMapping("/{userId}/{cartId}")
    public ResponseEntity<CartDto> createCart(@PathVariable Long userId, @PathVariable Long cartId) {
        CartDto newCartDto = new CartDto(cartId, userId);
        return ResponseEntity.ok(newCartDto);
    }
:)

Reply:
I don't know. I'm not sure how it works on frontend side. You have to pass information about user. You probably can do it both ways with JSON and with @PathVariable but I'm not 100%. For now I will just live it like that. 
Marcin: With Post you will not need to pass cartId information. cartId will be generated automatically in Cart class with  @GeneratedValue.